server/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "scripts": {
     "start": "node server.js",
-    "dev": "nodemon server.js",
-    "build:ui": "rm -rf build && cd ../client && npm run build --prod && cp -r build ../server"
```

Comment:
@JoeSangine Same question as my previous comment. Why is build commands being taken out to the server/package.json?

Reply:
see comment above